tests/test_animation.py

```diff
@@ -72,20 +76,52 @@ def test_send_all_args(self, bot, chat_id, animation_file, animation, thumb_file
         assert message.animation.file_name == animation.file_name
         assert message.animation.mime_type == animation.mime_type
         assert message.animation.file_size == animation.file_size
-        assert message.animation.thumb.width == 320
-        assert message.animation.thumb.height == 180
+        assert message.animation.thumb.width == self.width
+        assert message.animation.thumb.height == self.height
 
     @flaky(3, 1)
-    def test_resend(self, bot, chat_id, animation):
-        message = bot.send_animation(chat_id, animation.file_id)
+    @pytest.mark.timeout(10)
+    def test_get_and_download(self, bot, animation):
+        new_file = bot.get_file(animation.file_id)
+
+        assert new_file.file_size == self.file_size
+        assert new_file.file_id == animation.file_id
+        assert new_file.file_path.startswith('https://')
+
+        new_file.download('game.gif')
+
+        assert os.path.isfile('game.gif')
+
+    @flaky(3, 1)
+    @pytest.mark.timeout(10)
+    def test_send_mp3_url_file(self, bot, chat_id, animation):
```

Comment:
Please rename to `test_send_animation_url_file` ;)

Reply:
Fixed